dapps/marketplace/src/utils/pasteIntoInput.js

```diff
@@ -0,0 +1,17 @@
+export default function pasteIntoInput(el, text) {
+  el.focus()
+  if (
+    typeof el.selectionStart == 'number' &&
```

Comment:
nit: we use the === operator throughout our codebase. to be consistent, could you update ?
Same comment fo line 11 below.

Reply:
done